Sources/SafeDICore/Models/Scope.swift

```diff
@@ -78,27 +78,31 @@ final class Scope: Hashable {
 
     func createScopeGenerator(
         for property: Property? = nil,
-        instantiableStack: OrderedSet<Instantiable> = [],
         propertyStack: OrderedSet<Property> = []
     ) throws -> ScopeGenerator {
-        if let cycleIndex = instantiableStack.firstIndex(of: instantiable) {
-            throw ScopeError.dependencyCycleDetected([instantiable] + instantiableStack.elements[0...cycleIndex])
+        if
+            let property,
+            property.propertyType.isConstant,
+            let cycleIndex = propertyStack.firstIndex(of: property)
+        {
+            throw ScopeError.dependencyCycleDetected([property.typeDescription] + propertyStack.elements[0...cycleIndex].map(\.typeDescription))
```

Comment:
using property type descriptions instead of instantiables here isn't necessarily ideal since we no longer always reference the concrete instantiable type... but using the property type description makes it easier to find/follow the cycle in the code imo, so overall I think this is a win of a change